daisy/workflow/workflow.go

```diff
@@ -343,6 +381,27 @@ func (w *Workflow) nameToInstanceLink(n string) string {
 	return fmt.Sprintf("projects/%s/zones/%s/instances/%s", w.Project, w.Zone, n)
 }
 
+type gcsLogger struct {
```

Comment:
I've been trying to keep things in alphabetical order and categorized by their associated structs. e.g. step methods with step, resource methods with resource. You're a savage and put gcsLogger right in the middle of the (w *Workflow) methods.